### DISCUSSION TITLE
my copilot suddenly stopped working

### DISCUSSION THREAD

**Discussion by bayiremir** [community/community#108018] [Copilot Conversations]
### Select Topic Area

Bug

### Body

![Ekran Resmi 2024-02-12 13 06 03](https://github.com/community/community/assets/54590132/26794994-054a-4ae9-8a4f-4c1ef7d5d3c0)
like i said,It was working normally 2-3 hours ago. It has been like this for about 1 hour.

**Comment by Akash1134** [comment]
Hi @bayiremir ,

Sorry about the trouble caused. We are aware of the issue and our engineers are investigating it.

Thanks!

**Comment by florianchevallier** [comment]
![Capture d’écran 2024-02-12 à 11 56 01](https://github.com/community/community/assets/42930731/1d604c77-d598-49d5-aa19-104fa4e9a9b0)

Same here with a quota error. 

Writting it here to lead other people : ResourceRateLimitBy1s

**Comment by sovetski** [comment]
Same problem here

```
[error] [fetch] Unhandled status from server: 429 {"error":{"code":"429","message":"Requests to the Create a completion from a chosen model Operation under OpenAI Language Model Instance API have exceeded call rate limit of your current OpenAI S0 pricing tier. Please retry after 1 second. Please go here: https://aka.ms/oai/quotaincrease if you would like to further increase the default rate limit.","policy_id":"ResourceRateLimitBy1s"}}

[error] [fetch] Unhandled status from server: 504 request timeout

```

**Comment by hassni12** [comment]
Same problem here

**Comment by AhmedSaiedDiv** [comment]
same here

**Comment by Susomacias** [comment]
Same problem, but in my case sometimes it gives error 504 and sometimes it returns 0 results
![image](https://github.com/community/community/assets/85075478/8087441c-7428-4b98-ad89-f733049207a6)

**Comment by kero99** [comment]
same here

**Comment by robinmalik** [comment]
Dupe (looks like they're aware): https://github.com/orgs/community/discussions/108008

**Comment by Akash1134** [comment]
👀 Update - We are starting to see recovery based on the signals that the team have been monitoring, following mitigation steps being taken. When confident that recovery is complete, we will resolve this incident.

[Source](https://www.githubstatus.com/)

**Comment by Akash1134** [comment]
Hi everyone,

We are investigating reports that the earlier problem with Copilot code completions is reoccurring.

Source: [GitHub Status](https://www.githubstatus.com/)

_You may self troubleshoot some of the [common issues with Copilot](https://docs.github.com/en/copilot/troubleshooting-github-copilot/troubleshooting-common-issues-with-github-copilot#github-copilot-not-working-in-some-files) using our Docs._

Thanks!

**Reply by bayiremir** [reply]
still same problem rn

**Reply by a-kbv** [reply]
I still have the same problem

**Reply by PatrickHuynh97** [reply]
Likewise, inline suggestions worked briefly ~20 minutes ago, but have stopped working again.

**Reply by a-kbv** [reply]
![image](https://github.com/community/community/assets/67587336/8d56c319-b9c9-49c7-880e-5dd637ed4f68)

**Reply by abhinav-cashify** [reply]
Still facing same issue
<img width="1447" alt="image" src="https://github.com/community/community/assets/91053066/e7c82be8-2bfd-4c64-b816-26f8e27b305a">

**Reply by Akash1134** [reply]
We are investigating reports that the earlier problem with Copilot code completions is reoccurring.

**Comment by MKP1510** [comment]
Unhandled status from server: 429 {"error":{"code":"429","message":"Requests to the Create a completion from a chosen model Operation under OpenAI Language Model Instance API have exceeded call rate limit of your current OpenAI S0 pricing tier. Please retry after 1 second. Please go here: https://aka.ms/oai/quotaincrease if you would like to further increase the default rate limit.","policy_id":"ResourceRateLimitBy1s"}}


same error

**Reply by Akash1134** [reply]
We are investigating reports that the earlier problem with Copilot code completions is reoccurring.

**Comment by Akash1134** [comment]
We have confirmed that this is a reoccurrence of the earlier issue. Impact is currently limited to some European users. The team is working through alternative mitigation strategies to resolve the issue and return normal service.

**Reply by erikologic** [reply]
Getting 0 solutions:
```
2024-02-12 15:29:08.593 [info] [default] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
2024-02-12 15:29:08.692 [info] [default] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 99 ms
2024-02-12 15:29:08.693 [info] [streamChoices] solution 0 returned. finish reason: [stop]
2024-02-12 15:29:08.693 [info] [streamChoices] request done: headerRequestId: [7cd5b378-6be3-4837-a01d-47698f1c48d2] model deployment ID: [w1bad1925db0f]
```

**Comment by PatrickHuynh97** [comment]
Inline code suggestions are working for me now

**Comment by Akash1134** [comment]
Current Status as of Feb 12, 2024 - 15:16 UTC:

Following mitigation steps taken, we have reduced the impact to a more narrow subset of users. Investigation continues to completely mitigate the issue and restore Copilot code completion functionality.

**Comment by Akash1134** [comment]
#### Current Status as of Feb 12, 2024 - 15:51 UTC:


Code completions are now working for the impacted users, but completing more slowly than expected. Investigation continues to completely mitigate the issue and restore Copilot code completion functionality to normal.

**Comment by davecheney** [comment]
Duplicate of https://github.com/orgs/community/discussions/108008
